### PR TITLE
REST API

### PR DESCRIPTION
**Pull request by zJakeLoo**
Implement jQuery Ajax for REST call.
Feed a few fake clubs data.
Add clubs page which fetch the clubs detail from backend.
Change background image for header.
Change Title and Logo to 'ASDVC'
Remove user.go, as it is useless.
...

### CODE REVIEW

**Comment by Daniel-Awai** [comment]
Are you still getting the compiler error on your side?

**Comment by Daniel-Awai** [comment]
belay that.

File: frontend/static/clubs.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <link href="./css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+  <link href="./css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+</head>
+<body>
+  <nav class="teal lighten-1" role="navigation">
+    <div class="container">
+      <div class="nav-wrapper">
+        <a id="logo-container" href="#" class="brand-logo">Logo</a>
+        <ul id="nav-mobile" class="right side-nav">
+          <li><a href="/">Home</a></li>
```
Comment (Daniel-Awai) [inline frontend/static/clubs.html]: Add Getting Started
Add Forms

**Comment by Daniel-Awai** [comment]
After that :shipit:

**Comment by zJakeLoo** [comment]
Due to the "appengine" usage, I can't just compile the files locally. 
AppEngineSDK is required to be used for compile the files. 
Details will be given when we have group "meeting" or I will write down the guide.

File: frontend/static/clubs.html (hunk 1 above)
Comment (zJakeLoo) [inline frontend/static/clubs.html]: We will have to talk a little on this.

**Pull request merged**